gdrive/export_api.py

```diff
@@ -138,7 +138,7 @@ class FindModel(BaseModel):
     Request body format for the `/find` endpoint
     """
 
-    responseId: str
+    responseId: str | list[str]
```

Comment:
How heavy is the lift to move the responseId field to just be a list instead of a union field? We could avoid a few steps in the `/find` endpoint, and would be cleaner given your making a list out of the single value use case anyway

Reply:
i didn't want to break existing clients